### PR TITLE
Migration script

### PR DESCRIPTION
**Pull request by CalabashSquash**
# Changes
- Add `migration-script` folder, which has a python script which pulls from etherscan API then submits calls to `LentMycWithMigration.multiMigrate`.
- Add `multiMigrate` to `LentMycWithMigration.sol`. This allows batching of migrate calls.

### CODE REVIEW

File: migration-script/get_addresses.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      "outputs": [],
+      "stateMutability": "nonpayable",
+      "type": "function"
+    }]"""
+    print(_abi)
+    contract = w3.eth.contract(address=LENT_MYC_ADDRESS, abi=_abi)
+
+    i = 0
+    while i < len(addresses):
+        tx = contract.functions.multiMigrate(addresses[i:min(i + 30, len(addresses))]).buildTransaction(tx_skeleton(w3, PUBLIC_ADDRESS))
```
Comment (dospore) [inline migration-script/get_addresses.py:59]: upper bound exclusive, lower bound inclusive?

File: test/V2/migrationTest.t.sol [hunk 2]
```diff
@@ -95,6 +95,146 @@ contract Migration is Test {
...
+        vm.prank(migrator);
+        mycLendV2.multiMigrate(addressList);
+
+        // Claimable amount shouldn't change
+        claimableAmount1 = mycLendV2.getClaimableAmount(address(this));
+        assertApproxEqAbs(
+            claimableAmount1,
+            rewardAmount,
+            1 + mycLendV2.dust() / 2
+        );
```
Comment (dospore) [inline test/V2/migrationTest.t.sol:180]: would it be worth checking that `mycLend.getClaimableAmount` is as expected here as well?

File: migration-script/get_addresses.py (hunk 1 above)
Comment (CalabashSquash) [reply migration-script/get_addresses.py:59]: Yep

File: test/V2/migrationTest.t.sol (hunk 2 above)
Comment (CalabashSquash) [reply test/V2/migrationTest.t.sol:180]: Hmmmm I don't think so just because `address(mycLend) == address(mycLendV2)`, but I'll add it anyway.

**Review by doylio — APPROVED** [review]

**Pull request merged**